app/src/main/java/org/simple/clinic/patient/Patient.kt

```diff
@@ -83,14 +83,9 @@ data class Patient(
   fun withNameAndGender(fullName: String, gender: Gender): Patient =
       copy(fullName = fullName, gender = gender)
 
-  fun withoutAgeAndDateOfBirth(): Patient =
```

Comment:
I am guessing we removed this because there is no use case where we want a patient with age or dob?

Reply:
Yup.